docs/lib/content/commands/npm-config.md

```diff
@@ -33,13 +29,12 @@ npm config set key=value [key=value...]
 npm set key=value [key=value...]
 ```
 
-Sets each of the config keys to the value provided. Modifies the user configuration
-file unless [`location`](/commands/npm-config#location) is passed.
+Sets each of the config keys to the value provided.
```

Comment:
This is interesting, it added a new wrap here.

Reply:
I know there was some "wrap on sentences" discussion but I saw it inlining multiple sentences before.  Is this intentional? It's fine if it is I just wanted to call out out in case it was worth a second look.

Reply:
Upon further review it was my memory that was at fault, they were commas not periods.  All good.